### ISSUE TITLE
suggestion: accessibility  of elements on the main web admin interface needs improving

### ISSUE THREAD

**Issue by marrie** [getgrav/grav-plugin-admin#2233]
bug: when I am using my [screen reader](http://nvda-project.org) to navigate  the admin interface, I observe a lot of unlabeled links. For example the links for settings, dashboard, etc are not labeled. I only knwo what they are as the title bar changes. the big issue is the page creation. I accidently deleted all of my content when I was working on a page for the site. All of the items say "clickable" as I arrow through them. If you  want to test this yourself, download a copy of NVDA, it's free, and use the screen reader without looking at the scren, then turn on the focus rectangle to see what's going on and what I'm talking about. I found your platform through my web host and I want to actually try it to see if it is better than the bigger CMS platforms, as the more we can make CMS platforms accessible and inclusive, the more variety will be out there for web designers like myself. Plus, I love markdown for creation. If accessibility is taken seriously, I will be recomending this over the bigger cms platforms to my blind web designers.

**Comment by ndarilek** [comment]
Checkboxes/toggles are another big issue. I dug into this a while back and the admin theme seems to use some JavaScript library that replaces checkboxes/toggles with something that isn't a checkbox input. Thought about fixing that but I'm at capacity for accessibility work ATM. For now I'm just using git/VS code but it'd be nice if the admin interface was an option for me as well.
